aggregate_root_test.go

```diff
@@ -1,4 +1,4 @@
-package goengine_test
+package goengine
```

Comment:
just curious why changing to be the same package name? are you going to use this package internals in the test, otherwise would say `_test.go` with exposed package interface (see: https://medium.com/@matryer/5-simple-tips-and-tricks-for-writing-unit-tests-in-golang-619653f90742)